### PR TITLE
RDoc-1123 Client API > operations > maintenance > etl

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/maintenance/etl/reset-etl.dotnet.markdown [hunk 1]
```diff
@@ -1 +1,16 @@
-﻿TODO
+﻿# Operations : How to reset ETL?
+
+ETL process continues to send documents where it last finished. To reset pointer to last sent document and start from the beginning you should use **ResetEtlOperation**.
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/maintenance/etl/reset-etl.dotnet.markdown]: ETL is processing documents from the point where last batch finished. To start the processing from the very beginning you can reset the ETL by using the **ResetEltOperation**.

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/maintenance/etl/reset-etl.dotnet.markdown [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+## Syntax
+
+{CODE reset_etl_1@ClientApi\Operations\ResetEtl.cs /}
+
+| Return Value | | |
+| ------------- | ----- | ---- |
+| **configurationName** | string | ETL configuration name |
+| **transformationName** | string | Name of ETL transformation name |
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/maintenance/etl/reset-etl.dotnet.markdown]: Name of ETL transformation (drop last 'name')

Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/operations/maintenance/etl/reset-etl.dotnet.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/maintenance/etl/reset-etl.dotnet.markdown (hunk 1 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/operations/maintenance/etl/reset-etl.dotnet.markdown]: done

**Comment by ml054** [comment]
@ppekrol Please review my changes.

**Pull request merged**